dcos/dcos-ui
# DCOS-38492 delete job menu refactor

**SCOPE**

- Extract the JobDeleteModal into its own component
- Use the mediator pattern to delete jobs

**HOW TO TEST**

Scenario 1: Simple Job
Create a Job, and remove it. Try with some 3G slow to see if the modal states go fine

Scenario 2: Running Job
If the job is running, the modal should ask for extra confirmation.

Compare all these scenarios with a running cluster.

**WHAT WAS TAKEN OUT OF SCOPE**

- Not using graphql yet.
